test-app/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "createjs-accessibility-tester",
   "version": "1.0.0",
   "description": "Test and sample webapp for the Createjs Accessibility Module",
-  "main": "src/index.js",
+  "main": "src/index",
   "scripts": {
     "build": "rm -rf dist && NODE_ENV=production webpack",
-    "lint": "eslint --color src",
+    "lint": "eslint --color src --no-eslintrc -c ./.eslintrc.js",
```

Comment:
- eslint here was linting all the way up to the root and using its eslintrc, @mmoss localized it here to only `/test-aspp`